### ISSUE TITLE
Documentation comment blocks are a bit unreadable

### ISSUE THREAD

**Issue by DarkStoorM** [thomasabishop/alien-blood-vscode#2]
Hi again, the color on docBlocks is too bright, IMO, you can't read the selected text, not sure which entry this is exactly, but this applies to the documentation comments inserted from snippets, where VSCode selects a part of the text:

![img](https://user-images.githubusercontent.com/7021295/149638165-8beb4194-c30e-41e7-9237-e137c9131c30.png)

Manually selected text is basically unreadable:

![img](https://user-images.githubusercontent.com/7021295/149638307-49acba5d-0de0-4e4b-b49b-6ef2bdd585f4.png)

I noticed this behavior for:
- comment.block.documentation
- comment.line.double-slash
- comment.block

and probably applies to all comment-like sections

**Comment by thomasabishop** [comment]
Hey, that's a good call, thanks for raising it. Thing is I'm trying not to introduce too many new colours beyond the original terminal palette but in certain cases this isn't possible. I'll get this sorted soon. Cheers.

**Comment by thomasabishop** [comment]
@DarkStoorM Have changed the comment colour and reduced the opacity of the selection colour. See what you think.

**Comment by DarkStoorM** [comment]
@thomasabishop hmm, I don't know what the original Terminal scheme is, so I don't have any points of reference, but the punctuation is nice now

![img](https://user-images.githubusercontent.com/7021295/149662414-79bae559-3a71-42bf-9c73-6c1d22d22385.png)

But as a personal preference, I toned down the alpha a bit on the `editor.selectionBackground` (77) - I sometimes have to re-read selected comments while keeping some multiple selections active - I know you don't want to change the scheme this much, the current commit is fine too.

I used `#1d412577` for the `editor.selectionBackground`
![img](https://user-images.githubusercontent.com/7021295/149662572-fca559b9-bf4a-42cd-b8d9-ca59820ef039.png)

I also found out what was happening with the snippet highlights, these keys were missing:
```json
"editor.snippetFinalTabstopHighlightBackground": "#1d412544",
"editor.snippetTabstopHighlightBackground": "#1d412544",
```

I had to use very low alpha (44), because they blend with the `selectionBackground`. I can still read the block while it's highlighted, but I think it won't match the scheme you are going with
![img](https://user-images.githubusercontent.com/7021295/149662714-93707573-87c8-417f-bc83-c488511cdf17.png)

---

By the way, out of many dark themes, this one is my favorite, works out really well at night :D 👍

**Comment by thomasabishop** [comment]
Hey, apologies, I forgot to add the punctuation colour to the actual comment token. Have fixed that now in 0.1.11. I also prefer the alpha on your version of the selection highlight so have carried that and the tabstop colours over to the theme. Thanks for your input and the kind words. Still tweaking things but getting there :)

**Comment by thomasabishop** [comment]
@DarkStoorM Hey, apologie, I forgot to add the punctuation colour to the actual comment token. Have fixed that now in 0.1.11. I also prefer the alpha on your version of the selection highlight so have carried that and the tabstop colours over to the theme. Thanks for your input and the kind words. Still tweaking things but getting there :)

**Comment by DarkStoorM** [comment]
@thomasabishop Looks great now 👍 I think we have this issue resolved now 🎉 good work
